lib/Basics/debugging.h

```diff
@@ -156,7 +161,7 @@ struct is_container
               !std::is_same_v<unsigned char*, typename std::decay<T>::type> &&
               !std::is_same_v<unsigned char const*,
                               typename std::decay<T>::type> &&
-              !std::is_same_v<T, std::string> &&
+              !is_basic_string<T>::value &&
               !std::is_same_v<T, std::string_view> &&
               !std::is_same_v<T, const std::string>,
```

Comment:
Could be removed if we remove const from T above

Reply:
Discussed with @MBkkt : can be ignored for now.